Dockerfile

```diff
@@ -3,41 +3,43 @@ FROM python:3.7.3-slim as base
 ENV AIRFLOW_HOME=/airflow
 ENV AIRFLOW__CORE__DAGS_FOLDER=${AIRFLOW_HOME}/dags
 ENV AIRFLOW__CORE__BASE_LOG_FOLDER=${AIRFLOW_HOME}/logs
-ENV AIRFLOW__CORE__EXECUTOR=LocalExecutor
+ENV AIRFLOW__CORE__EXECUTOR=CeleryExecutor
 ENV AIRFLOW__CORE__PLUGINS_FOLDER=${AIRFLOW_HOME}/plugins
 ENV AIRFLOW__CORE__CHILD_PROCESS_LOG_DIRECTORY=${AIRFLOW_HOME}/logs/scheduler
 
 ENV AIRFLOW__CORE__SQL_ALCHEMY_CONN=postgresql+psycopg2://postgres:example@db/airflow-db
 ENV AIRFLOW__CORE__LOAD_EXAMPLES=false
+ENV AIRFLOW__CORE__DAGS_ARE_PAUSED_AT_CREATION=false
+
+ENV AIRFLOW__CELERY__BROKER_URL=sqla+${AIRFLOW__CORE__SQL_ALCHEMY_CONN}
+ENV AIRFLOW__CELERY__RESULT_BACKEND=db+${AIRFLOW__CORE__SQL_ALCHEMY_CONN}
 
 WORKDIR ${AIRFLOW_HOME}
 
-COPY ./requirements ./requirements
+COPY ./requirements.txt ./
 
 RUN pip install -U pip \
     && set -ex \
     && apt-get update -yq \
     && apt-get install -yq --no-install-recommends build-essential \
-    && pip install --no-cache-dir -r ./requirements/base.txt \
+    && pip install --no-cache-dir -r ./requirements.txt \
     && apt-get remove --purge --autoremove -yq build-essential \
     && rm -rf ~/.cache/* \
     && rm -rf /var/lib/apt/lists/* \
     && rm -rf /tmp/*
 
+COPY ./dags ${AIRFLOW__CORE__DAGS_FOLDER}
+
 FROM base as dev
 
 ENV PYTHONUNBUFFERED=true
 ENV PYTHONDONTWRITEBYTECODE=true
 
-RUN pip install --no-cache-dir -r ./requirements/dev.txt \
-    && rm -rf ~/.cache/* \
-    && rm -rf /tmp/*
+ENV AIRFLOW__CORE__EXPOSE_CONFIG=true
 
 ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.5.0/wait /wait
```

Comment:
alternative is to run `wget` or `curl` inside a single `RUN` to retrieve this, but it's probably not available in the image and it makes little difference